sdk/redis/Azure.ResourceManager.Redis/CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Release History
 
+## 1.3.0 (2023-04-01)
+
+### Features Added
+
+- Enable Persistence(aof/rdb) on storage account in a different subscription than the cache via Managed Identity Auth Type. RedisConfiguration has a new property storage-subscription-id.
+
+### Bugs Fixed
+
+- Updated validation to ensure access to storage account while enabling persistence.
+
 ## 1.2.0-beta.2 (Unreleased)
```

Comment:
Change this unreleased `1.2.0-beta.2` to stable `1.2.0` and set the date to latest date.
`1.3.0 (2023-04-01)` The date is earlier than the previous release date, and the version skips `1.2.0` stable.